test/org/sagebionetworks/bridge/validators/ScheduleValidatorTest.java

```diff
@@ -25,11 +26,12 @@ public void before() {
     }
     
     @Test
-    public void mustHaveAtLeastOneActivity() {
+    public void mustHaveAtLeastOneActivityAndAScheduleType() {
```

Comment:
Add unit tests for:
* schedule with both cron and interval
* expires too short